### PR TITLE
url is Invalid

### PR DESCRIPTION
**Pull request by baitercel**
this "http://www.dmoz.org/Computers/Programming/Languages/Python/Books/" is  403  ,  so  l change url is https://www.resource-zone.com/    , please merge my  submit

### CODE REVIEW

File: dirbot/spiders/dmoz.py [hunk 1]
```diff
@@ -6,10 +6,10 @@
 
 class DmozSpider(Spider):
     name = "dmoz"
-    allowed_domains = ["dmoz.org"]
+    allowed_domains = ["resource-zone"]
```
Comment (baitercel) [inline dirbot/spiders/dmoz.py]: sorroy ,  "allowed_domains = ["resource-zone.com"]" is right

**Comment by baitercel** [comment]
* add "main.py"  to  quick start
* change "www.dmoz.org" to "dmoztools.net"

**Comment by redapple** [comment]
Hello @baitercel ,
I'm afraid this repository is now deprecated. Please see the [deprecation notice in the README](https://github.com/scrapy/dirbot/blob/master/README.rst).
The Scrapy team will not accept patches to it anymore.
An equivalent education scrapy project is now at https://github.com/scrapy/quotesbot

**Comment by baitercel** [comment]
l see , think u

**Pull request closed without merging**